demo/vmdefs/centos7-mezzanine/Vagrantfile

```diff
@@ -0,0 +1,29 @@
+VAGRANTFILE_API_VERSION = "2"
```

Comment:
let's create a similar concept like for http vagrant-boxes-enabled and vagrant-boxes-disabled, or some specific way where we can easily specify what boxes we want to start. Right now we would be starting 4 vagrant boxes and we would need cca 16GB RAM per instance.